buddyboss/buddyboss-platform
# PROD-914 #1492 - Fix: Sort Group Types Alphabetically

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1492 .

### How to test the changes in this Pull Request:

1. Enable Group Type from BuddyBoss > Settings > Groups
2. Add some Group Types and enable this option: **Display this group type in "Types" filter in Groups Directory**
3. Go to Website > Groups
4. Can see the Group Type filter with alphabetical order

### Proof Screenshots or Video
https://vimeo.com/624350869/0db62ef33c


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
